### PR TITLE
feat(ui): fixed the dropdown styles and added the sub-dropdown

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Added the sub dropdowns and updated the background color with our theme colours.

### Changes

- Updated the header.
- Updated the dropdown with theme colour.

## Screenshots
Screenshot from 2021-06-25 13-09-13
Screenshot from 2021-06-25 13-09-18

### CODE REVIEW

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working fine.

**Comment by Shruti3004** [comment]
@GMishx Resolved the merge conflicts.

File: src/api/auth.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const fetchTokenApi = (username, password) => {
   return sendRequest({
     url,
     method: "POST",
-    credentials: "include",
+    credentials: false,
```
Comment (GMishx) [inline src/api/auth.js:33]: Do we really need this change?

Comment (Shruti3004) [reply src/api/auth.js]: @GMishx No not necessarily, Ihave fixed it

**Pull request merged**